packages/legend-extension-dsl-persistence/src/graph-manager/protocol/pure/v1/model/packageableElements/persistence/V1_DSL_Persistence_ServiceOutput.ts

```diff
@@ -27,11 +28,15 @@ export abstract class V1_ServiceOutput implements Hashable {
 }
 
 export class V1_GraphFetchServiceOutput extends V1_ServiceOutput {
+  keys!: object[];
+  path!: object;
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.